### PR TITLE
Jh/remove one any

### PR DESCRIPTION
**Pull request by joelhooks**
[previous PR was merged rapidly]

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export type Resource = {
```
Comment (joelhooks) [inline src/types.ts:1]: type makes more sense if we aren't using classes I think?

File: src/pages/lessons/index.tsx [hunk 2]
```diff
@@ -1,52 +0,0 @@
-import Link from 'next/link'
```
Comment (joelhooks) [inline src/pages/lessons/index.tsx:1]: these are actually hard forwarded to `/search` so never displayed

File: src/types.ts (hunk 1 above)
Comment (johnlindquist) [reply src/types.ts:1]: Yes. `Interface` is strictly for OOP patterns IMO.

File: now.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "redirects": [
-    {"source": "/lessons", "destination": "/s"},
-    {"source": "/courses", "destination": "/s"}
+    {"source": "/lessons", "destination": "/s?type=lesson"},
```
Comment (johnlindquist) [inline now.json:3]: This is starting to remind me of how codesandbox does links:

http://codesandbox.io/s/vanilla
http://codesandbox.io/s/github
http://codesandbox.io/s/vue

etc, etc, etc.

**Pull request merged**